src/mobile/mobile.go

```diff
@@ -110,12 +110,14 @@ func (m *Yggdrasil) StartJSON(configjson []byte) (conduit *dummy.ConduitEndpoint
 }
 
 // Stop the mobile Yggdrasil instance
-func (m *Yggdrasil) Stop() error {
+func (m *Yggdrasil) Stop() {
+	logger := log.New(m.log, "", 0)
```

Comment:
Would be better to factor this out into the `Yggdrasil` struct so we reuse a single logger, rather than creating more of them.